link_linux.go

```diff
@@ -1012,18 +1012,12 @@ func (h *Handle) LinkAdd(link Link) error {
 	return h.linkModify(link, unix.NLM_F_CREATE|unix.NLM_F_EXCL|unix.NLM_F_ACK)
 }
 
-// LinkModify modify an existing link device. The type and features of the device
-// are taken from the parameters in the link object.
-// Equivalent to: `ip link add $link`
 func LinkModify(link Link) error {
```

Comment:
Any reason for removing the go doc comment ?

Reply:
the master doesn't have the comments ,,rebasing removed it